roles/kubernetes-apps/ansible/tasks/kubedns.yml

```diff
@@ -8,10 +8,10 @@
     - { name: kube-dns, file: kubedns-sa.yml, type: sa }
     - { name: kube-dns, file: kubedns-deploy.yml, type: deployment }
     - { name: kube-dns, file: kubedns-svc.yml, type: svc }
-    - { name: kubedns-autoscaler, file: kubedns-autoscaler-sa.yml, type: sa }
```

Comment:
Do all 4 of the resources we're renaming need to be cleaned up here? Trying to recall if the kube module overides the name in the file.